src/ansible_midi.c

```diff
@@ -351,8 +336,14 @@ void handler_MidiFrontLong(s32 data) {
 ////////////////////////////////////////////////////////////////////////////////
 ///// common cv utilities
 
-inline static uint16_t pitch_cv(u8 num, s16 offset) {
-	return SEMI14[num] + offset;
+inline static int clamp(int n, int lower, int upper) {
```

Comment:
fwiw the `sclip` (signed clip/clamp) function in libavr32 is the same as `clamp` here (minus the inlining)

Reply:
...I must have subconsciously done this to ensure symmetry with [my nearly identical remark on your PR from the other day](https://github.com/monome/norns/pull/1156#discussion_r450005568). Fixed in `215706f`